sound/soc/intel/boards/sof_maxim_common.h

```diff
@@ -12,33 +12,16 @@
 
 #include <sound/soc.h>
 
-#define MAX_98373_CODEC_DAI	"max98373-aif1"
-#define MAX_98373_DEV0_NAME	"i2c-MX98373:00"
-#define MAX_98373_DEV1_NAME	"i2c-MX98373:01"
-
-extern struct snd_soc_dai_link_component max_98373_components[2];
-extern struct snd_soc_ops max_98373_ops;
-extern const struct snd_soc_dapm_route max_98373_dapm_routes[];
```

Comment:
`max_98373_dapm_routes` is also used by sof_sdw_max98373.c, please edit it, too.
```
sound/soc/intel/boards/sof_sdw_max98373.c:51:45: error: ‘max_98373_dapm_routes’ undeclared (first use in this function); did you mean ‘maxim_dapm_routes’?
   51 |  ret = snd_soc_dapm_add_routes(&card->dapm, max_98373_dapm_routes, 2);
      |                                             ^~~~~~~~~~~~~~~~~~~~~
      |                                             maxim_dapm_routes
```

Reply:
@bardliao , thanks reminder , will do.